src/isaac_ros/isaac_ros_launch/launch/nvblox.launch.py

```diff
@@ -60,7 +71,12 @@ def generate_launch_description():
     load_composable_nodes = LoadComposableNodes(
         target_container=component_container_name_arg,
         composable_node_descriptions=[
-            ComposableNode(name="nvblox_node", package="nvblox_ros", plugin="nvblox::NvbloxNode")
+            ComposableNode(
+                name="nvblox_node",
+                package="nvblox_ros",
+                plugin="nvblox::NvbloxNode",
+                remappings=getZedRemappings(),
```

Comment:
@BergerKing333 You still need to use the Condition "setup_for_zed" to determine if the remappings are needed or not, look at the code you deleted for the remappings, it was using a Condition.

Reply:
I'm not sure why this part was changed tbh, I thought the namespace was the only issue?

Reply:
I can make the remappings themselves be conditional, but this is how nvidia does it now. It doesn't work with the Remap function that we used to have, so we're doing it slightly different now. We should literally never be using nvblox without the zed anyways so im not sure why its a problem though. I only added one line though i just made the function more readable by dividing it out

Reply:
We use nvblox without a ZED in gazebo, so this is needed